core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraSession.kt

```diff
@@ -332,15 +339,24 @@ private fun createVideoUseCase(
     targetFrameRate: Int,
     stabilizationMode: StabilizationMode,
     dynamicRange: DynamicRange,
+    videoQuality: VideoQuality,
     backgroundDispatcher: CoroutineDispatcher
 ): VideoCapture<Recorder> {
     val sensorLandscapeRatio = cameraInfo.sensorLandscapeRatio
-    val recorder = Recorder.Builder()
+    val recorderBuilder = Recorder.Builder()
         .setAspectRatio(
             getAspectRatioForUseCase(sensorLandscapeRatio, aspectRatio)
         )
-        .setExecutor(backgroundDispatcher.asExecutor()).build()
-    return VideoCapture.Builder(recorder).apply {
+        .setExecutor(backgroundDispatcher.asExecutor())
+    if (videoQuality != VideoQuality.DEFAULT) {
+        recorderBuilder.setQualitySelector(
+            QualitySelector.from(
+                VideoQuality.toQuality(videoQuality)!!,
+                FallbackStrategy.higherQualityOrLowerThan(VideoQuality.toQuality(videoQuality)!!)
+            )
+        )
+    }
```

Comment:
Try to avoid using `!!`. Use something like:
```
val recorder = Recorder.Builder()
    .setAspectRatio(
        getAspectRatioForUseCase(sensorLandscapeRatio, aspectRatio)
    )
    .setExecutor(backgroundDispatcher.asExecutor())
    .apply {
        VideoQuality.toQuality(videoQuality)?.let { quality ->
            setQualitySelector(QualitySelector.from(quality, ...))
        }
    }.build()
```